docs/deployment/user-management.md

```diff
@@ -19,83 +19,10 @@ Object.entries(frontMatter).filter(
-5. Click **Create**.
-
-You have successfully created a Semgrep organization that you can now connect to GitHub or GitLab.
-
-### Connecting a Semgrep organization account to your SCM
-
-This step connects your Semgrep organization account to a source code manager, such as GitHub. Linking a source code manager allows the Semgrep org's membership to be managed by GitHub or GitLab. After connecting your Semgrep org to GitHub or GitLab, members from GitHub or GitLab are able to join the Semgrep organization you created and connected.
-
-You can only connect your Semgrep organization to the source code manager that you originally logged in with. For example, if you logged in to Semgrep Cloud Platform with GitHub, you do not see the **Connect to GitLab** option. If your organization uses both GitHub and GitLab to manage source code, log in with the source code manager that you would prefer to use to manage Semgrep org membership. You can still scan repositories from other sources.
-
-:::info Connecting to GitHub
-When connecting to GitHub, Semgrep prompts you to install the Semgrep GitHub app for your organization. If you don't have permission to install apps on your GitHub organization, you won't be able to complete the connection. Identify a GitHub organization admin or owner who has permissions to install or approve the app before proceeding.
-:::
-
-1. Sign in to Semgrep Cloud Platform.
-2. On the sidebar, click **your account name** > **the organization account** you created.
-3. Click **Settings** > **Source Code Managers**.
-4. Click on your Source code manager, for example, **Connect to GitHub**.
-5. Follow the prompts in the Cloud Platform and select an organization or group to link.
-6. After a successful link, you are signed out of Semgrep Cloud Platform automatically, as your credentials have changed after linking an organization.
-7. Sign back in to Semgrep Cloud Platform.
-
-You have successfully connected an org in Semgrep Cloud Platform with an organization in your source code management tool.
-
-### Joining a Semgrep organization
-
-If a Semgrep organization has been connected to GitHub or GitLab, members of the org in GitHub or GitLab can join the Semgrep organization. To join a Semgrep org account, perform the following steps:
-
-1. Sign in to Semgrep Cloud Platform.
-2. On the sidebar, click **your account name > Add org**.
-3. Choose **Join an organization**. A list of possible orgs to join appears.
-4. Click the organization you want to join. You are signed out of Semgrep Cloud Platform automatically as your credentials change after linking an organization.
-6. Sign back in to Semgrep Cloud Platform.
-
-You have successfully joined an org in Semgrep Cloud Platform.
-
-## Controlling access through roles 
-
 Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role.
 
 ![Screenshot of role-based access control](/img/rbac-overview.png)<br />
```

Comment:
Can we update this screenshot?

Reply:
Thanks for the reminder. I get blind to these things.